### DISCUSSION TITLE
You exceeded you're current quota

### DISCUSSION THREAD

**Discussion by Eyobs-droid** [lobehub/lobehub#2654] [Self Hosting]
After finally deploying it using vercel, the following json pops up
![Uploading Screenshot from 2024-05-25 15-29-46.png…]()

**Comment by arvinxx** [comment]
image is not showed

**Comment by Eyobs-droid** [comment]
{
  "error": {
    "message": "You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors.",
    "type": "insufficient_quota",
    "param": null,
    "code": "insufficient_quota"
  },
  "provider": "openai",
  "endpoint": "https://api.openai.com/v1"
}

**Reply by arvinxx** [reply]
I think it's just you need to pay for api key
